src/main/java/io/supertokens/webserver/InputParser.java

```diff
@@ -239,7 +239,7 @@ public static Integer parseIntOrThrowError(JsonObject element, String fieldName,
 
     }
 
-    public static List<String> collectAllMissingRequiredFieldsOrThrowError(JsonObject input, List<String> requiredFields)
+    public static void collectAllMissingRequiredFieldsOrThrowError(JsonObject input, List<String> requiredFields)
```

Comment:
this function needs to be renamed given that we aren';t returning anything from it

Reply:
done